src/components/Photo/DocumentAutoCapture.js

```diff
@@ -14,16 +14,9 @@ const maxAttempts = 3
 
 const serverError = { name: 'SERVER_ERROR', type: 'error' }
 
-type Capture = {
```

Comment:
We were storing `id`, `base64`, `valid`, and `processed`... But as far as I can tell, we only ever leverage the `id` to determine whether our capture upload has completed or not. We don't use anything else.